Raven.NET.Core/Subjects/RavenSubject.cs

```diff
@@ -14,7 +14,7 @@ namespace Raven.NET.Core.Subjects
     public class RavenSubject
     {
         internal List<IRaven> Observers = new();
-        private Guid UniqueId { get; set; }
+        internal Guid UniqueId { get; set; }
```

Comment:
Why this change needed?

Reply:
Thought some logs may use having Id logged. In case something goes wrong we would know in which subject the problem appeared. If that's not necessary I'll remove it. I'm fine both ways

Reply:
Nah its fine